File: app/src/androidTest/java/com/kkaysheva/ituniver/java/InstrumentationTestSuite.java
package com.kkaysheva.ituniver.java;

import org.junit.runner.RunWith;
import org.junit.runners.Suite;

@RunWith(Suite.class)
@Suite.SuiteClasses({ContactsListTest.class,
        ContactTest.class, ContactsMapTest.class, ContactMapTest.class})
public class InstrumentationTestSuite {
}

File: app/src/androidTest/java/com/kkaysheva/ituniver/kotlin/InstrumentationTestSuite.kt
package com.kkaysheva.ituniver.kotlin

import org.junit.runner.RunWith
import org.junit.runners.Suite

@RunWith(Suite::class)
@Suite.SuiteClasses(ContactsListTest::class, ContactTest::class, ContactsMapTest::class, ContactMapTest::class)
class InstrumentationTestSuite {
}